tests/rt.conf

```diff
@@ -123,21 +123,23 @@ RUN     | fv3_ccpp_rrfs_v1beta
 
-COMPILE | CCPP=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta,FV3_GFS_2017_RRTMGP                                                                | standard    | hera.intel     | fv3         |
+COMPILE | CCPP=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta,FV3_GFS_v15p2_RRTMGP,FV3_GFS_v16beta_RRTMGP                                        | standard    | hera.intel     | fv3         |
 
 RUN     | fv3_ccpp_gfs_v15p2                                                                                                             | standard    | hera.intel     | fv3         |
 RUN     | fv3_ccpp_gfs_v16beta                                                                                                           | standard    | hera.intel     | fv3         |
-RUN     | fv3_ccpp_rrtmgp                                                                                                                | standard    | hera.intel     | fv3         |
+RUN     | fv3_ccpp_gfs_v15p2_RRTMGP                                                                                                      | standard    | hera.intel     | fv3         |
+RUN     | fv3_ccpp_gfs_v16beta_RRTMGP                                                                                                    | standard    | hera.intel     | fv3         |
 
 #######################################################################################################################################################################################
 # CCPP DEBUG tests                                                                                                                                                                     #
 #######################################################################################################################################################################################
 
-COMPILE | CCPP=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta,FV3_GFS_2017_RRTMGP DEBUG=Y                                                        | standard    | hera.intel     | fv3         |
+COMPILE | CCPP=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta,FV3_GFS_v15p2_RRTMGP,FV3_GFS_v16beta_RRTMGP DEBUG=Y                                | standard    | hera.intel     | fv3         |
 
 RUN     | fv3_ccpp_gfs_v15p2_debug                                                                                                       | standard    | hera.intel     | fv3         |
 RUN     | fv3_ccpp_gfs_v16beta_debug                                                                                                     | standard    | hera.intel     | fv3         |
-RUN     | fv3_ccpp_rrtmgp_debug                                                                                                          | standard    | hera.intel     | fv3         |
+RUN     | fv3_ccpp_gfs_v15p2_RRTMGP_debug                                                                                                | standard    | hera.intel     | fv3         |
```

Comment:
I assume I owe you the four regression test files in `tests/tests`, and the removal of the old ones: 
```
fv3_ccpp_gfs_v15p2_RRTMGP
fv3_ccpp_gfs_v16beta_RRTMGP
fv3_ccpp_gfs_v15p2_RRTMGP_debug
fv3_ccpp_gfs_v16beta_RRTMGP_debug
```

Reply:
@climbfuji 
I added them.